### PR TITLE
Add release workflow

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+           npm version ${{ inputs.release-version }} -m "Release %s"
+
+           sed -i 's/Unreleased/${{ inputs.release-version }}/g' CHANGELOG.md
+           git add CHANGELOG.md
+           git commit --amend --date=now --no-edit
+
+       - name: Build server
+         run: |
+           export JAVA_TOOL_OPTIONS=--add-opens=java.base/java.util.concurrent=ALL-UNNAMED
+           cd vividus-studio-server && ./mvnw clean install -DskipTests
```
Comment (ikalinin1) [inline .github/workflows/release.yml:42]:
```suggestion
           cd vividus-studio-server
           ./mvnw clean install -DskipTests
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+name: VIVIDUS Studio Release
+
+ on:
+   workflow_dispatch:
+     inputs:
+       release-version:
+         required: true
+         type: string
+         description: The release version
```
Comment (valfirst) [inline .github/workflows/release.yml:9]: add example on input, like `1.0.0`

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+     runs-on: ubuntu-latest
+
+     steps:
+       - name: Checkout repository
+         uses: actions/checkout@v3
+         with:
+           token: ${{ secrets.GH_RELEASE_TOKEN }}
+
+       - name: Install JDK 17
+         uses: actions/setup-java@v3.9.0
```
Comment (valfirst) [inline .github/workflows/release.yml:22]:
```suggestion
         uses: actions/setup-java@v3

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+         with:
+           distribution: 'temurin'
+           java-version: 17
+           cache: 'maven'
+
+       - name: Create release commit
+         run: |
+           git config --global user.email "uarlouski@gmail.com"
```
Comment (valfirst) [inline .github/workflows/release.yml:30]: epam.com ?

**Review by valfirst — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 4 above)
Comment (uarlouski) [reply .github/workflows/release.yml:30]: I use gmail for everything :)

File: .github/workflows/release.yml (hunk 1 above)
Comment (uarlouski) [reply .github/workflows/release.yml:42]: just to save execution time, the tests are guaranteed to be successful at this point

**Review by ikalinin1 — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**